tensorflow/contrib/learn/python/learn/estimators/estimator.py

```diff
@@ -1203,12 +1211,6 @@ def _get_eval_ops(self, features, labels, metrics):
     model_fn_ops = self._call_model_fn(
         features, labels, model_fn_lib.ModeKeys.EVAL)
```

Comment:
you need to send metrics to call_model_fn, right?
could you please add a test which catches this error?